projects/plugins/jetpack/_inc/lib/markdown/extra.php

```diff
@@ -2597,7 +2598,8 @@ function _doTable_leadingPipe_callback($matches) {
 		return $this->_doTable_callback(array($matches[0], $head, $underline, $content));
 	}
 	function _doTable_callback($matches) {
-		$head		= $matches[1];
+		$attr = [];
+  $head		= $matches[1];
```

Comment:
Another indentation WTF.

This time the insertion makes sense, except since we don't phpcs this file it didn't pick up the use of `[]` instead of `array()`.